spec/models/pac_bio_run_spec.rb

```diff
@@ -52,7 +52,8 @@
               'tag2_identifier' => '1',
               'tag2_sequence' => 'GTCA',
               'tag2_set_id_lims' => '2',
-              'tag2_set_name' => 'tag_set_2'
+              'tag2_set_name' => 'tag_set_2',
+              'pipeline_id_lims' => 'IsoSeq'
```

Comment:
I'd expect this to appear twice in the tests - once in input test data, and once checking it comes out the other end somewhere? Probably misunderstanding the test though. :)

Reply:
Not really, its just mostly letting rails handle the specifics, and is mostly assuming that if an attribute is defined, then rails will let it get to the database. Maybe it is assuming a bit to much of the glue works. (Although the glue itself is unit tested)